From 49bb708ffda4be9cb28a04d83096d8aab244430e Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Fri, 14 Jun 2013 14:30:39 -0700
Subject: [PATCH 140/974] gpio: gpio-octeon Make Kconfig depend on
 CAVIUM_OCTEON_SOC

The GPIO unit is associated with some SoCs, it is not part of the CPU,
so the Kconfig must reflect this.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/gpio/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index 652e3ea..33d1e92 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -192,7 +192,7 @@ config GPIO_MXS
 
 config GPIO_OCTEON
 	tristate "Cavium OCTEON GPIO"
-	depends on GPIOLIB && CPU_CAVIUM_OCTEON
+	depends on GPIOLIB && CAVIUM_OCTEON_SOC
 	default y
 	help
 	  Say yes here to support the on-chip GPIO lines on the OCTEON
-- 
2.6.2

